js/Genoverse.js

```diff
@@ -318,14 +318,18 @@ const Genoverse = Base.extend({
 
     for (i = 0; i < config.length; i++) {
       track = tracksById[config[i].id];
-      trackFromLibrary = tracksFromLibraryById[config[i].id];
+      trackFromLibrary = tracksFromLibraryById[config[i].defaultId].extend();
 
       if (track) {
         setConfig(track, config[i]);
         track._fromStorage = true;
       }
       else if(trackFromLibrary) {
         trackFromLibrary._fromStorage = true;
+        trackId = config[i].defaultId;
+
+        this.trackIds          = this.trackIds          || {};
+        this.trackIds[trackId] = this.trackIds[trackId] || 1;
```

Comment:
This part was added because same kind of tracks where having same id. That way there was only one tracks of kind were showing. This fixes ability to add multiple of same